docs/tutorial/workers.md

```diff
@@ -224,7 +258,7 @@ prefect deployment run 'get_repo_info/my-deployment'
 ```
 
 !!! danger "Common Pitfalls"
-    - When running `prefect deploy`, double check that you are at the **root of your repo**, otherwise the worker may attempt to use an incorrect flow entrypoint during remote execution!
+    - Keep and run your deploy scripts at the **root of your repo**, otherwise the built Docker file may be missing files that it needs to execute!
```

Comment:
```suggestion
    - Store and run your deploy scripts at the **root of your repo**, otherwise the built Docker file may be missing files that it needs to execute!
```